perllib/FixMyStreet/App/Controller/Admin/ManifestTheme.pm

```diff
@@ -0,0 +1,97 @@
+package FixMyStreet::App::Controller::Admin::ManifestTheme;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+use FixMyStreet::App::Form::ManifestTheme;
+
+sub auto :Private {
+    my ($self, $c) = @_;
+
+    if ( $c->cobrand->moniker =~ /^default|fixmystreet$/ ) {
```

Comment:
No working site will have `default` as their moniker, though. Perhaps if the length of `available_cobrand_classes` is equal to 1 rather than the default check? I realise still need `fixmystreet` (or something) to be the "parent" cobrand.

Reply:
If there's only one cobrand available then wouldn't it be better to bypass the list page and only allow editing of that single cobrand's theme?

Reply:
Yes, that's what I meant, sorry if I wasn't clear - instead of checking for 'default' check if available is equal to 1 and do the same.